apps/emqx_conf/etc/emqx_conf.md

```diff
@@ -2,6 +2,20 @@ EMQ X configuration file is in [HOCON](https://github.com/emqx/hocon) format.
 HOCON, or Human-Optimized Config Object Notation is a format for human-readable data,
 and a superset of JSON.
 
+## Layered
+
+EMQ X configuration consists of 3 layers.
+From bottom up:
+
+1. Immutable base: `emqx.conf` + `EMQX_` prfixed environment variables.<br>
+   Changes in this layer require a full node restart to take effect.
+1. Cluster override: The path of which is configured by `cluster_override_conf_file`.<br>
```

Comment:
Do we plan to use include(include "cluster-override.conf"), or configure(cluster_override_conf_file)?

Reply:
Include will make ENV, not in the base layer.  Definitely the configure file will be used. :0